### ISSUE TITLE
Future: Timer to show when the next chapter will be uploaded

### ISSUE THREAD

**Issue by Warewolf007** [KotatsuApp/Kotatsu#1146]
### Describe your suggested feature

A feature that can show timer or countdown of  
when the next chapter of a manga or manhwa will be uploaded. it well help many users rather than waiting for a single manga or manhwa and can give time to others. 
Please think about it, It is a great idea.

Thank you 
very much

### Acknowledgements

- [X] This is not a duplicate of an existing issue. Please look through the list of [open issues](https://github.com/KotatsuApp/Kotatsu/issues) before creating a new one.

**Comment by Mhuzaifakhalid** [comment]
*upvote*

**Comment by MariusAlbrecht** [comment]
have you thought about where that information is going to come from?
In my experience that kind of information is very rarely directly provided by sources.

**Comment by Mhuzaifakhalid** [comment]
> have you thought about where that information is going to come from?
> In my experience that kind of information is very rarely directly provided by sources.

It doesnt have to be exact, it can just provide an estimate as new chapters are either weekly or monthly

**Comment by galpt** [comment]
nice one! most manga/manhwa series release a new chapter after 1 week and looks like it's almost a common rule to do that for manga creators. so a countdown that uses the chapter release `Date` and use that to countdown till next week is simple enough to implement imho

**Comment by Warewolf007** [comment]
> This also could be a calendar to show when the next chapter of the manga you have usually comes

Yes that's it. it could be a celender that shows one' favourite manga or anything's up coming date not exact but a slight hint of time like (the manga's next chapter well release in between 1pm or 7pm in --/--/---- date) like that or (you can think something of your own that is user friendly and easy to do).

Thank you for replying
I appreciate it

**Comment by Warewolf007** [comment]
> have you thought about where that information is going to come from? In my experience that kind of information is very rarely directly provided by sources.

I also think the same, but think if we find a way than kotatsu well be the one that have something that others don't, I am not saying that all the source should have this feature but the most used one's can have it, and i think the creators can find a way to do so of the information of upcoming chapter from the source.

Thank you for replying
But I am sorry for not 
being Helpful😔

**Comment by Warewolf007** [comment]
> > It doesnt have to be exact, it can just provide an estimate as new chapters are either weekly or monthly

Yes it doesn't have to be exact, A day in which the upcoming chapter is released can do the thing.

Thank you for replying 
I am greatful that you have 
acknowledged my Idea👍
